Source/Message/MessageClient+Sync.swift

```diff
@@ -30,8 +30,8 @@ extension MessageClient {
     /// - parameter beforeMessage: List messages sent before a message by id.
     /// - parameter max: Limit the maximum number of messages in the response.
     /// - returns: Messages array
-    public func list(roomId roomId: String, before: String? = nil, beforeMessage: String? = nil, max: Int? = nil) throws -> [Message] {
-        return try SyncUtil.getArray(roomId, before, beforeMessage, max, async: list)
+    public func list(roomId: String, before: String? = nil, beforeMessage: String? = nil, max: Int? = nil) throws -> [Message] {
+        return try SyncUtil.getArray(roomId, before, beforeMessage, max, async: list(roomId:before:beforeMessage:max:queue:completionHandler:))
```

Comment:
The original name postToRoom & postToPerson is just to avoid compile ambiguous error. 
If you want to add the parameter labels to avoid the error, I suggest that we change the name from "postTo" to "post", so that consist with all other API name.